frontend/packages/pipelines-plugin/src/utils/pipeline-augment.ts

```diff
@@ -210,9 +210,15 @@ export const getPipelineFromPipelineRun = (pipelineRun: PipelineRunKind): Pipeli
   };
 };
 
-export const getTaskStatus = (pipelinerun: PipelineRunKind): TaskStatus => {
+const totalPipelineRunTasks = (pipelinerun: PipelineRunKind): number => {
   const executedPipeline = getPipelineFromPipelineRun(pipelinerun);
   const totalTasks = (executedPipeline?.spec?.tasks || []).length ?? 0;
+  const finallyTasks = (executedPipeline?.spec?.finally || []).length ?? 0;
```

Comment:
`spec` is not an optional field in the `pipeline`, but this `getPipelineFromPipelineRun` may return `null` when the pipelinerun status is not present initially when it is created. So for better readability, we could be explicit here instead of optional chaining, What do you think?
eg: 
```suggestion
  if ( !executedPipeline ) {
   return 0;
} else {
  const totalTasks = (executedPipeline.spec?.tasks || []).length ?? 0;
  const finallyTasks = (executedPipeline.spec?.finally || []).length ?? 0;
  return totalTasks + finallyTasks;
}
```

Reply:
Updated.